languages/csharp/exercises/concept/operator-overloading/OperatorOverloadingTests.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Xunit;
+
+public class OperatorOverloadingTests
+{
+    [Fact]
+    public void Equality_true()
+    {
+        Assert.True(Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(55, "HD")));
+    }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void Equality_false()
+    {
+        Assert.False(Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(60, "HD")));
```

Comment:
Maybe also add tests for the `!=` operator?

Reply:
Please see comment about incompleteness of `OperatorOverloadingTests.cs`.